doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+.. _NEP48:
+
+=====================================
+NEP 48 — Spending NumPy Project Funds
+=====================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Author: Inessa Pawson <inessa@albuscode.org>
+:Author: Stefan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Informational
+:Created: 2021-02-07
+:Resolution:
+
+
+Abstract
+--------
+
+The NumPy project has historically never received significant **unrestricted**
+funding. However, that is starting to change.  This NEP aims to provide
+guidance about spending NumPy project unrestricted funds, by formulating a set
+of principles about *what* to pay for and *who* to pay. It will also touch on
+how decisions regarding spending funds get made, how funds get administered,
+and transparency around these topics.
+
+
+Motivation and Scope
+--------------------
+
+In its 16+ year history, the NumPy project has only spent on the order of
+$10,000 USD of funds that were not restricted to a particular program.  Project
```

Comment:
What is a program?

Reply:
It’s another commonly used term in non-profit and academic funding. Would the word “purpose” be less confusing?

Reply:
Since we expect this document to live for many years, I think rough dates would be more appropriate:
```
Since its inception and until 2020, the NumPy project has only seen about $10,000 of unrestricted* funding.

* unrestricted funding is money contributed to the project with no conditions for use: the project can decide to spend it on equipment, salary, conferences, or any other purpose as the project sees fit.
```